### PR TITLE
📖  Update footer per project formation

### PR DESCRIPTION
**Pull request by afmarcum**
Update footer to meet requirements for finalizing LF project formation.

### CODE REVIEW

File: scorecards-site/modules/Footer/Footer.vue [hunk 1]
```diff
@@ -12,7 +12,9 @@
           ]"
       /></a>
       <div class="text-14 mb-12">
-        © 2022 The Linux Foundation, under the terms of the Apache License 2.0.
+        Copyright © OpenSSF Scorecard a Series of LF Projects, LLC
+        <br />
+        For web site terms of use, trademark policy and other project policies please see https://lfprojects.org.
```
Comment (justaugustus) [inline scorecards-site/modules/Footer/Footer.vue:17]:
```suggestion
        For website terms of use, trademark policy and other project policies please see https://lfprojects.org.
```

I think this should be one word, but a quick check with LF Legal would be good.

**Review by justaugustus — COMMENTED** [review]
@afmarcum — one nit here!

File: scorecards-site/modules/Footer/Footer.vue (hunk 1 above)
Comment (afmarcum) [reply scorecards-site/modules/Footer/Footer.vue:17]: @justaugustus Message sent to LF Legal on whether we can change to "website" and add missing commas.

Comment (spencerschrock) [reply scorecards-site/modules/Footer/Footer.vue:17]: Can we also make the link a link?

**Review by spencerschrock — APPROVED** [review]

**Review by justaugustus — APPROVED** [review]
Thanks for getting this updated, @afmarcum!

**Pull request merged**